pkg/dev/podmandev/reconcile.go

```diff
@@ -118,6 +123,20 @@ func (o *DevClient) reconcile(
 	return nil
 }
 
+// warnAboutK8sResources prints a warning if the Devfile contains a K8s resource that it needs to create on Podman.
+func (o *DevClient) warnAboutK8sResources(devfileObj parser.DevfileObj) {
+	k8sComponents, err := devfile.GetKubernetesComponentsToPush(devfileObj, false)
+	if err != nil {
+		return
+	}
+	var compNames []string
+	for _, comp := range k8sComponents {
+		compNames = append(compNames, comp.Name)
+	}
+	log.Warningf("Skipping Kubernetes component %q. Kubernetes components are not supported on Podman.", strings.Join(compNames, ", "))
```

Comment:
This should only be displayed if `compNames` is not empty.

Reply:
Small suggestion: I'd also reword the warning message like so, to have the reason displayed first:
```
 ⚠  Kubernetes components are not supported on Podman. Skipping "$comp1", "$comp2".
```

What do you think?